data/src/test/java/org/apache/iceberg/TestMergingMetrics.java

```diff
@@ -37,6 +48,7 @@
 
 import static org.apache.iceberg.types.Types.NestedField.optional;
 import static org.apache.iceberg.types.Types.NestedField.required;
+import static org.junit.Assert.assertEquals;
```

Comment:
Nit: the community prefers to avoid static method imports. There are places (usually my fault) that still import optional and required like the lines above, but we tend to discourage doing this in new pull requests. Could you remove this and just import `Assert`?